Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -425,6 +428,22 @@ class TabTrayController: UIViewController, UITabBarDelegate, UICollectionViewDel
         view.addSubview(navBar)
     }
 
+    private func relayoutTabs() {
+        numberOfColumns = numberOfColumnsForCurrentSize()
```

Comment:
I think we might be able to remove this method and call `collectionView.invalidateLayout` instead.